main.go

```diff
@@ -0,0 +1,11 @@
+package http_wasm
```

Comment:
let's name it as a single word following the convention
```suggestion
package httpwasm
```

Reply:
sure I'll change that all around